src/library_browser.js

```diff
@@ -292,7 +292,8 @@ var LibraryBrowser = {
           ctx = GL.getContext(contextHandle).GLctx;
         }
         // Set the background of the WebGL canvas to black
-        canvas.style.backgroundColor = "black";
+        if (!contextAttributes["alpha"])
+          canvas.style.backgroundColor = "black";
```

Comment:
Actually, come to think about it, I can't even recall why we set the background color to black? Was this to make alpha blending somehow behave more expected?